api/src/main/java/org/openmrs/api/context/UserContext.java

```diff
@@ -453,4 +456,40 @@ private void setUserLocation() {
 			}
 		}
 	}
+	
+	/**
+   * Notifies privilege listener beans about any privilege check.
+   * <p>
+   * It is called by {@link UserContext#hasPrivilege(java.lang.String)}.
+   * 
+   * @see PrivilegeListener
+   * @param user the authenticated user or <code>null</code> if not authenticated
+   * @param privilege the checked privilege
+   * @param hasPrivilege <code>true</code> if the authenticated user has the required privilege or
+   *            if it is a proxy privilege
+   * @since 1.8.4, 1.9.1, 1.10
+   */
+  @Transactional(readOnly = true)
```

Comment:
Given that the method is now private, I believe this is now redundant

Reply:
part of: https://github.com/openmrs/openmrs-core/pull/2699/commits/82aecd86b9d917b4443f24a7ca8ddbf6c886a4d8